examples/llama2/requirements.txt

```diff
@@ -1,6 +1,7 @@
 datasets>=2.8.0
-git+https://github.com/huggingface/optimum.git
+optimum>=1.17.0
 onnx>=1.14.0
 protobuf==3.20.2
 torch -i https://download.pytorch.org/whl/nightly/cu118
 transformers>=4.33.2
+auto-gptq
```

Comment:
Please also add the pass and it's dependencies (optimum, auto-gptq) to https://github.com/microsoft/Olive/blob/9e7eba8c8d3fd500a0d0240210472c05169f17d2/olive/workflows/run/run.py#L47